### DISCUSSION TITLE
インシデントの重複却下、インシデント同士の関連付けができるようにしたい

### DISCUSSION THREAD

**Discussion by gr1m0h** [topotal/waroom-roadmap#69] [Feature Request]
https://github.com/topotal/waroom-roadmap/issues/45 に近いが、過去の類似インシデントとの関連というよりは、同時期に複数のアラートが発生し、自動作成により複数のWaroomインシデントが作成されてしまった際に、１つのインシデントに情報を集約したい

そのため、重複するインシデントやインシデント同士の関連付けが欲しい
また、重複却下の場合はSlackに通知しないようにしたい

**Comment by sawa-zen** [comment]
@gr1m0h 
要望ありがとうございます！

もし Datadog の Integration から起票されている場合は aggregation_key をアラートに設定していただくことで一つのインシデントとして扱うことができ、Resolve のアラートから自動的にインシデントステータスを Resolve にすることも可能です。
https://docs.waroom.com/04bf0cd4488048c0ad7f684540f226d6

ただまとめられたアラートの一覧や手動での変更、AI によるアラートを跨いだ紐づけなどまだまだ足りない機能があると思うのでこちら検討していきます！
